src/plugins/controls/public/control_group/embeddable/control_group_container_factory.ts

```diff
@@ -40,7 +40,9 @@ export class ControlGroupContainerFactory implements EmbeddableFactoryDefinition
   public isEditable = async () => false;
 
   public readonly getDisplayName = () => {
-    return ControlGroupStrings.getEmbeddableTitle();
+    return i18n.translate('controls.controlGroup.title', {
+      defaultMessage: 'Control group',
+    });
```

Comment:
Nice 👍

Reply:
I figure you got what this change did, but I'll write it down here in case anyone else is looking:

Removing the reference to the strings file here makes it so that the entire strings file _doesn't_ end up in the page load bundle.